### PR TITLE
write event stdout in the event emit callback

### PR DESCRIPTION
**Pull request by ryanpetrello**

### CODE REVIEW

**Comment by ryanpetrello** [comment]
attempts to address: https://github.com/ansible/ansible-runner/issues/182

File: ansible_runner/utils.py [hunk 1]
```diff
@@ -326,6 +306,19 @@ def _emit_event(self, buffered_stdout, next_event_data=None):
             event_data['end_line'] = self._start_line + n_lines
             self._start_line += n_lines
             if self._event_callback:
+                if event_data.get('stdout'):
+                    if self.output_json:
+                        stdout_actual = json.dumps(event_data)
```
Comment (AlanCoding) [inline ansible_runner/utils.py]: In the case that `output_json` is true, do you need to skip the case of `stdout_actual != "{}"`? It seems that that's a bit of logic that may have been missed. But then again, events with blank lines are extremely common, and I have no idea what the expectations could possibly be for those with that setting.

**Comment by matburt** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck tarball jobs fixed

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by matburt** [comment]
recheck

**Comment by ryanpetrello** [comment]
Haven't found time for this, might reinvestigate later.

**Pull request closed without merging**